lib/index.js

```diff
@@ -117,7 +117,7 @@ module.exports = function (source) {
   // let isTS = false
   const { script, scriptSetup } = descriptor
   if (script || scriptSetup) {
-    const lang = script.lang || (scriptSetup && scriptSetup.lang)
+    const lang = script?.lang || scriptSetup?.lang
```

Comment:
```suggestion
    const lang = (script && script.lang) || (scriptSetup && scriptSetup.lang)
```

Thanks for the quick fix!
However, v15's source code is untranspiled JavaScript and may run in environments that do not support optional chaining syntax. So here we must use the logical OR operator instead.